package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@transcend-io/penumbra",
+  "sideEffects": false,
   "version": "4.4.10",
```

Comment:
Can you bump to 4.5.0

Reply:
Yeah. Not quite ready to merge. There's a file with a side effect that tree shaking is busting